src/transactions/typeTag/parser.ts

```diff
@@ -50,6 +54,7 @@ type TypeTagState = {
 
 export enum TypeTagParserErrorType {
   InvalidTypeTag = "unknown type",
+  InvalidButPossiblyGeneric = "unknown type. Did you mean to allow generics?",
```

Comment:
Full types must start with 0x..., so we can just say it's an unexpected generic.

```suggestion
  InvalidButPossiblyGeneric = "unexpected generic.  Did you mean to allow generics?",
```